app/src/org/commcare/dalvik/activities/CommCareWiFiDirectActivity.java

```diff
@@ -630,61 +621,6 @@ public void onFailure(int reason) {
         });
     }
 
-    @Override
-    public void disconnect() {

```

Comment:
Seems a little strange that this isn't really ever called (only from within `cancelDisconnect`, which also isn't ever called)